ui-components/src/actions/Button.tsx

```diff
@@ -4,25 +4,26 @@ import { AppearanceProps, classNamesForAppearanceTypes } from "../global/Appeara
 import { Icon, IconSize, UniversalIconType } from "../icons/Icon"
 
 export interface ButtonProps extends AppearanceProps {
-  id?: string
-  type?: "button" | "submit" | "reset"
+  "data-test-id"?: string
```

Comment:
Maybe it would be better to keep props written as a camelCase and use `data-test-id` for HTML elements only, what do you think?

Reply:
Super down! This is existing and would be a bigger change bc most of our buttons use the `data-test-id` syntax but I 100% thing we should do this esp considering that we also have `dataTestId` as a prop on this component 😬